src/content/docs/zh-cn/guides/upgrade-to/v4.mdx

```diff
@@ -0,0 +1,391 @@
+---
+title: 升级到 Astro v4
+description: 如何将你的项目升级到 Astro 的最新版本 (v4.0)。
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+import FileTree from '~/components/FileTree.astro'
+
+本指南将帮助你从 Astro v3 迁移到 Astro v4。
+
+需要将旧项目升级到 v3 吗？请参阅我们的 [旧版本迁移指南](/zh-cn/guides/upgrade-to/v3/)。
+
+需要查阅 v3 的文档？请访问这个[旧版本文档站 (不被维护的 v3.6 快照)](https://docs-git-v3-docs-unmaintained-astrodotbuild.vercel.app/)。
+
+## 升级 Astro
+
+使用你的包管理器将项目的 Astro 和所有官方集成升级到最新版本。
+
+<PackageManagerTabs>
+  <Fragment slot="npm">
+  ```shell
+  # 同时升级 Astro 和官方集成
+  npx @astrojs/upgrade
+  ```
+  </Fragment>
+  <Fragment slot="pnpm">
+  ```shell
+  # 同时升级 Astro 和官方集成
+  pnpm dlx @astrojs/upgrade
+  ```
+  </Fragment>
+  <Fragment slot="yarn">
+  ```shell
+  # 同时升级 Astro 和官方集成
+  yarn dlx @astrojs/upgrade
+  ```
+  </Fragment>
+</PackageManagerTabs>
+
+
+如果需要的话，你也可以[手动升级 Astro 集成](/zh-cn/guides/integrations-guide/#manual-upgrading)，你可能还需要升级项目中的其他依赖。
```

Comment:
Ah, we have the same problem here as we do for the pt-BR pr... that is a NEW section on the integrations guide that probably hasn't been translated yet.

So your options are to:
- remove the link entirely
- just link to the main page (not the header, which is new and doesn't exist on the zh-cn site
- link to the *English* version of the site in order to link to that exact section.

I'll let you decide what you'd prefer to do @Genteure !

Reply:
I'll link them to the English doc for now, so we can quickly find them later by searching `(/en/` and replace them when those pages are updated.

Reply:
> Link to unexpected language in src/content/docs/zh-cn/guides/upgrade-to/v4.mdx, line 41: Expected link path to start with "/zh-cn/", but found "/en/". The correct prefix is required to ensure that users stay on their selected language version of the docs.

Uhh, link checker is not happy those links.